MODULE_NOT_FOUND error on api start

## Description
My goal was to run the API on my local device but I didn't find any instructions. Quite probably I missed something. In that case I'd recommend step-by-step guides in the sub-directories `README.md`-files.

## Current Behavior
MODULE_NOT_FOUND error after executing `npm run build` in `/api`. Terminal output:
```
> api@1.0.0 start
> node ./build/server.js

node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module '@lib/config'
Require stack:
- /.../WalletPOC/api/build/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/.../WalletPOC/api/build/server.js:33:34)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/.../WalletPOC/api/build/server.js' ]
}

Node.js v18.11.0
```

Also, TypeScript errors after executing `npm start` in `/api`. Terminal output:
```
> api@1.0.0 build
> tsc -p tsconfig.json && tsc-alias

routes/user/user.repository.ts:14:3 - error TS2322: Type 'User & { keyShares: MpcKeyShare[]; }' is not assignable to type 'User'.
  Types of property 'keyShares' are incompatible.
```

## Expected Behavior
No errors and executable js files in `/api/build`.

## Steps to Reproduce
Branch: `master`
1. `git clone git@github.com:lauhon/WalletPOC.git`
2. `cd ./WalletPOC/database`
3. `docker-compose up -d` (no errors)
4. `cd ../api`
5. `npm ci` (no errors)
6. `npm run db:reset` (no errors, db live and tables created as defined in `/api/prisma/schema.prisma`)
7. `npm run build`

## System & Dependencies
* macOS Ventura 13.0.1 (22A400)
* Xcode 14.1 Build version 14B47b
* Node.js v18.11.0
* npm 8.19.2

--- a/api/routes/circle/circle.service.ts
+++ b/api/routes/circle/circle.service.ts
@@ -212,7 +212,7 @@ const findUserByPaymentCard = (payment: CirclePaymentResponse): ResultAsync<Paym
 const findUsdcAddress = (user: User): ResultAsync<string, RouteError> => {
   const ethereumAddressShare = user.keyShares.find((share) => share.path === config.ethereumAddressPath);
 
-  if (ethereumAddressShare?.address === undefined)
+  if (!ethereumAddressShare || ethereumAddressShare.address === null)
     return errAsync(other("No matching Ethereum Address could be found"));
 
   return okAsync(ethereumAddressShare.address);

--- a/api/routes/user/wallet.d.ts
+++ b/api/routes/user/wallet.d.ts
@@ -2,5 +2,5 @@ export interface MpcKeyShare {
   id: string;
   path: string;
   keyShare: string;
-  address?: string;
+  address: string | null;
 }

--- a/api/routes/user/wallet.repository.ts
+++ b/api/routes/user/wallet.repository.ts
@@ -11,7 +11,7 @@ export const deleteWallet = (wallet: MpcKeyShare) => {
   });
 };
 
-export const createDerivedWallet = (user: User, share: string, path: string): Promise<MpcKeyShare> => {
+export const createDerivedWallet = async (user: User, share: string, path: string): Promise<MpcKeyShare> => {
   return client.mpcKeyShare.create({
     data: {
       keyShare: share,

